File: INICIANTE/1043/Main.java

import java.util.Locale;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in).useLocale(Locale.ENGLISH);
        double a = scanner.nextDouble();
        double b = scanner.nextDouble();
        double c = scanner.nextDouble();
        boolean isTriangle = a + b > c && a + c > b && b + c > a;
        if (isTriangle) {
            System.out.printf("Perimetro = %.1f\n", a + b + c);
        } else {
            System.out.printf("Area = %.1f\n", (a + b) * c / 2);
        }
        scanner.close();
    }
}

File: INICIANTE/1043/main.cpp
#include <iomanip>
#include <iostream>
using namespace std;

int main() {
    float a, b, c;
    cin >> a >> b >> c;
    bool isTriangle = a + b > c && a + c > b && b + c > a;
    if (isTriangle)
        cout << "Perimetro = " << fixed << setprecision(1) << a + b + c << endl;
    else
        cout << "Area = " << fixed << setprecision(1) << (a + b) * c / 2 << endl;
    return 0;
}